<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>741</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>잔고 및 보유종목현황</string>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>621</width>
       <height>61</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>1</number>
     </property>
     <row/>
     <column>
      <property name="text">
       <string>예수금</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총매입금액</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총평가금액</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총평가손익금액</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총수익률(%)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>추정자산</string>
      </property>
     </column>
    </widget>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>721</width>
       <height>221</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>1</number>
     </property>
     <row/>
     <column>
      <property name="text">
       <string>종목번호</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>종목명</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>매입가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>평가손익</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>수익률(%)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>보유수량</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>매매가능수량</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>현재가</string>
      </property>
     </column>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>644</x>
       <y>23</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>실시간조회</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>50</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>조회</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>291</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>접속정보</string>
    </property>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>271</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>343</y>
      <width>311</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>보유종목 상승/하강률</string>
    </property>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="geometry">
      <rect>
       <x>134</x>
       <y>159</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>실시간조회</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>19</y>
       <width>291</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>156</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>조회</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>152</y>
      <width>291</width>
      <height>521</height>
     </rect>
    </property>
    <property name="title">
     <string>종목조회</string>
    </property>
    <widget class="QTableWidget" name="tableWidget_3">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>20</y>
       <width>271</width>
       <height>441</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>종목코드</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>종목명</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>203</x>
       <y>477</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>조회</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>471</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>코스피</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>492</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>코스닥</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1116</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
